### PR TITLE
Allow use of bundler

### PR DESCRIPTION
**Pull request by igas**
This is almost 1:1 to @szechyjs PR #43, except I've added the test. A failing example can be seen [here](https://github.com/igas/action-rubocop/runs/3267680470?check_suite_focus=true)

### CODE REVIEW

**Comment by igas** [comment]
@mgrachev any chance you'll have time to take a look at this?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: CI
+on: [pull_request]
+jobs:
+  dotenv-linter:
```
Comment (mgrachev) [inline .github/workflows/ci.yml:4]:
```suggestion
  test-skip-install-and-use-bundler:
```

**Review by mgrachev — COMMENTED** [review]
@igas I'm so sorry for the long answer. I was on vacation 😅

**Review by mgrachev — APPROVED** [review]

**Comment by review-dog** [comment]
Hi, @igas! We merged your PR to reviewdog! 🐶
Thank you for your contribution! ✨

**We just invited you to join the @reviewdog organization on GitHub.**
Accept the invite by visiting https://github.com/orgs/reviewdog/invitation.
By joining the team, you'll be a part of reviewdog community and can help the maintainance of reviewdog.

Thanks again!

**Pull request merged**